index.md

```diff
@@ -372,6 +372,17 @@ Here are some examples of messages and how the parameters would be represented a
 
 As these examples show, a trailing parameter (a final parameter with a preceding `':'`) has the same semantics as any other parameter, and MUST NOT be treated specially or stored separately once the `':'` is stripped.
 
+### Compatibility with incorrect software
+
+Servers SHOULD handle single `\r` or `\n` character as if it was a `\r\n` pair, to support existing clients that might send this. However, clients and servers alike MUST NOT send single `\r` or `\n` characters.
```

Comment:
I'm not super sure whether a lone `\r` is treated like this, would be good to test to make sure if ya' haven't already. Lone `\n` definitely is though, by all the servers.

Reply:
I haven't

Reply:
Solanum and Unreal support `\r`, Insp and Ergo don't.

Reply:
I agree, I'd prefer if "servers MAY handle single `\r`" instead of SHOULD.